message_hub/src/main/java/ordina/youbionic/YoubionicApplication.java

```diff
@@ -0,0 +1,28 @@
+package ordina.youbionic;
+
+//import jakarta.annotation.PreDestroy;
```

Comment:
I commented it out because it's something I didn't yet get to work properly; this feature branch is to automatically start and stop the Python files, but it also has some small fixes for previous features to make sure they work properly. I'm still figuring out how to do version management the right way :)